tests/e2e/visual/messages/messages-overview.wdio-spec.js

```diff
@@ -0,0 +1,131 @@
+const commonPage = require('@page-objects/default/common/common.wdio.page');
+const loginPage = require('@page-objects/default/login/login.wdio.page');
+const messagesPage = require('@page-objects/default/sms/messages.wdio.page');
+const dataFactory = require('@factories/cht/generate');
+const utils = require('@utils');
+const { generateScreenshot, resizeWindowForScreenshots, isMobile } = require('@utils/screenshots');
+const modalPage = require('@page-objects/default/common/modal.wdio.page');
+
+describe('Messages Overview', () => {
+  const docs = dataFactory.createHierarchy({
+    name: 'Visual Messaging User',
+    user: true,
+    nbrClinics: 5,
+    nbrPersons: 8,
+    useRealNames: true,
+  });
+
+  const seedIncomingSms = async (from, text, id = 'visual-a') => {
+    await utils.request({
+      method: 'POST',
+      path: '/api/sms',
+      body: { messages: [ { from, content: text, id } ] }
+    });
+  };
```

Comment:
```suggestion
  const seedIncomingSms = async (contact, text, id = 'visual-a') => {
    const smsRecord = {
      type: 'data_record',
      from: contact.phone,
      contact: {
        _id: contact._id,
        parent: contact.parent
      },
      sms_message: {
        message: text,
        from: contact.phone
      },
      reported_date: new Date().valueOf()
    };
    
    await utils.saveDoc(smsRecord);
  };
```

We need to add the contact to the sms record, so the screenshots show real names instead of numbers, which is why it is better to save the record directly.